query_ranges_to_vnodes.cc

```diff
@@ -107,3 +113,6 @@ void query_ranges_to_vnodes_generator::process_one_range(size_t n, dht::partitio
         add_range(get_remainder());
     }
 }
+
+template class query_ranges_to_vnodes_generator<dht::partition_range_vector>;
+template class query_ranges_to_vnodes_generator<dht::chunked_partition_range_vector>;
```

Comment:
Seems like bloat.

Reply:
That's because dht::partition_range_vector isn't converted across the board to chunked_vector.
Just in areas susceptible to long partition_range vectors.

Reply:
@avikivity I'm not sure I understand your comment.
Would you prefer to convert partition_range_vector to use chunked_vector unconditionally?

Reply:
> @avikivity I'm not sure I understand your comment. Would you prefer to convert partition_range_vector to use chunked_vector unconditionally?

ping @avikivity

Reply:
I don't think we need chunked_partition_range_vector. Where we have such large vectors, it's due to another, underlying problem.